app/models/Service.ts

```diff
@@ -35,6 +35,7 @@ export interface Service {
   resource: Organization;
   schedule: Schedule;
   source_attribution: string;
+  status: string;
```

Comment:
```suggestion
  status: 'pending' | 'approved' | 'rejected' | 'inactive';
```

If you feel comfortable with it, do you think we could turn this into a union of string literals? This both serves as documentation on the frontend side and it ensures that we don't attempt to compare it to an invalid string value (I think, at least; we might have to turn on some TypeScript or ESLint options for that).

Reply:
Great suggestion! I have added this. I tested this by comparing a `status` prop against a junk string; the TS compiler complained, so i think we're good.